README.md

```diff
@@ -40,47 +40,57 @@ cf set-env conjur-service-broker SECURITY_USER_PASSWORD [value]
-- `CONJUR_VERSION`: The version of your Conjur instance (`4` or `5`); defaults to 5.
-- `CONJUR_ACCOUNT`: The account name for the Conjur instance you are connecting to.
-- `CONJUR_APPLIANCE_URL`: The URL of the Conjur appliance instance you are connecting to. If using high availability, this should be the URL for the Master host in the cluster. This is the URL that the service broker will use to communicate with Conjur.
+- `CONJUR_VERSION`: the version of your Conjur instance (`4` or `5`); defaults to 5.
+- `CONJUR_ACCOUNT`: the account name for the Conjur instance you are connecting to.
+- `CONJUR_APPLIANCE_URL`: the URL of the Conjur appliance instance you are connecting to.
 - `CONJUR_FOLLOWER_URL` (HA only): If using high availability, this should be the URL for a load balancer that manages the cluster's Follower instances. This is the URL that applications that bind to the service broker will use to communicate with Conjur.
-- `CONJUR_POLICY`: The Policy where new Hosts should be added - the Conjur account specified in `CONJUR_AUTHN_LOGIN` needs `create` and `update` privilege on this Policy.
-  - `CONJUR_POLICY` is optional, but is strongly recommended. By default, if this value is not specified, Hosts will be added to the `root` Conjur policy, and the Conjur account that the Service Broker uses to manage the Hosts will need `create` and `update` privileges on the `root` Conjur policy.
-- `CONJUR_AUTHN_LOGIN`: The identity of a Conjur Host (of the form `host/host-id`) with `create` and `update` privileges on `CONJUR_POLICY`. This account will be used to add and remove Hosts from Conjur policy as apps are deployed to or removed from PCF.
+- `CONJUR_POLICY`: the Policy where new Hosts should be added - the Conjur account specified in `CONJUR_AUTHN_LOGIN` needs `create` and `update` privilege on this Policy.
+  > **NOTE:** The `CONJUR_POLICY` is optional, but is strongly recommended. By default, if this value is not specified, Hosts will be added to the `root` Conjur policy, and the Conjur account that the Service Broker uses to manage the Hosts will need `create` and `update` privileges on the `root` Conjur policy.
+
+  > **NOTE:** If you use multiple CloudFoundry foundations, this policy should include an identifier for the foundation to distinguish applications deployed in each. For example, if you have both a `production` and `development` foundation, then your policy branches for the Conjur Service Broker might be `cf/prod` and `cf/dev`.
+
+  > **NOTE:** If you are using v4 Conjur, the Service Broker requires your `CONJUR_POLICY` to have a Host Factory called `CONJUR_POLICY-apps`. For example, if your `CONJUR_POLICY` is `cf/prod`, you can add a Host Factory by updating your  policy file to include the following:
+  > ```yaml
+  > - !policy
+  >   id: cf
+  >   body:
+  >     - !policy prod
+  >       owner: !group cf-admin-group
+  >       body:
+  >       - !layer cf-apps
```

Comment:
actually, from [here](https://github.com/cyberark/conjur-service-broker/blob/master/lib/conjur_client.rb#L14) it expects the layer to have fully qualified ID `CONJUR_POLICY/CONJUR_POLICY-apps`

so it should be `!layer cf/prod/cf/prod-apps` ... which is not very pretty. I am glad this is v4 only.

Reply:
I updated this to reflect what I believe should be correct based on the link you provided.

It looks like this is for the host factory rather than the layer. And it doesn't have to be absolute in the policy file, but the ID does need to include the entire policy path.

Let me know if I'm missing something still.

Reply:
This looks right. For consistency I prefer to advise that the hf layer have the same id, but since it's not technically required I leave it up to you

Reply:
Sure thing, that makes a lot of sense!